app/models/user.rb

```diff
@@ -117,6 +117,20 @@ def self.search_all(query)
     users += search_groups(terms)
     users += search_users(terms)
     users.flatten
+    # binding.pry
+    # User.joins(:cohort).joins(:roles).joins(:groups)
+    #     .where(["upper(cohorts.name) iLIKE ANY (array[?])",
+    #             "upper(roles.name) iLIKE ANY (array[?])",
+    #             "upper(groups.name) iLIKE ANY (array[?])",
+    #             "upper(first_name) iLIKE ANY (array[?]) OR
+    #             upper(last_name) iLIKE ANY (array[?])"])
+    #
+    # User.joins(:cohort)
+    # .where("upper(name) iLIKE ANY (array[?])",query).
+    # joins(:roles).where("upper(name) iLIKE ANY (array[?])", terms)
+    # .joins(:groups).where("upper(name) iLIKE ANY (array[?])", terms)
+    # .where("upper(first_name) iLIKE ANY (array[?]) OR
+    #    upper(last_name) iLIKE ANY (array[?])", terms, terms)
```

Comment:
Do we want to keep these queries here? If we don't have a good reason, could we remove em?

Reply:
Sorry, I thought I had taken them out. Wouldn't want them in staging/master.